src/hockeypuck/server/settings.go

```diff
@@ -1,6 +1,6 @@
 /*
    Hockeypuck - OpenPGP key server
-   Copyright (C) 2012-2014  Casey Marshall
+   Copyright (C) 2012-2025 Hockeypuck Contributors
```

Comment:
I've been standardising on "Casey Marshall and the Hockeypuck Contributors", on existing files at least.